### ISSUE TITLE
Also export `gaverstehfest` as alias to `postwilder`.

### ISSUE THREAD

**Issue by lrnv** [jlapeyre/InverseLaplace.jl#24]
Hi, 

Would it be possible to also export a `gaverstehfest` method that would be an alias for the current `postwilder` ? I was looking for this algorithm and was about to start a fork to implement it because I did not saw it. 

In a large part of the literature, the name of this algorithm is 'Gaver-Stehfest', in the names of Gaver, who found the approximating series, and Stehfest, who found the accelerated convergence tricks. Post-Wilder are the ones that found the continuous formula that Gaver's work derives from. 

Also, the principal cited source in `postwilder.jl` is written by Stehfest, not Post-Wilder.

I do not ask for a change of name -- would probably be breaking --, an alias would be enough.

**Comment by heltonmc** [comment]
Thanks for bringing attention to this because the method does have competing naming. In my field it is generally referred to as the "Post-Widder" formula whereas I have seen many instances where "Gaver-Stehfest" is preferred. I'm not for sure adding two methods for this would lead to less confusion. It might be easiest to do this on the  user end if they prefer one name over the other by defining `const stehfest = postwid` in their module after importing the library. It should be mentioned that this is a relatively recent introduction to the package so it does not appear in the docs. 

A name change would be fine and fairly easy if we think that `stehfest` is the consensus name in the field. That paper is definitely cited a lot. I definitely don't want to be the one to assign credit to an algorithm because it kind of becomes a slippery slope on how far back something is derived from. I'm not for sure the Stehfest paper should be referred to as the principle source. I think the most exhaustive would be something like:

[1] Post, Emil L. "Generalized differentiation." Transactions of the American Mathematical Society 32.4 (1930): 723-781.
[2] Widder, David V. "The inversion of the Laplace integral and the related moment problem." Transactions of the American Mathematical Society 36.1 (1934): 107-200.
[3] T. (1943). The Laplace Transform. By D. V. Widder. Pp. x, 406. 36s. 1941. Princeton Mathematical series, 6. (Princeton University Press; Humphrey Milford). The Mathematical Gazette, 27(273), 37-39. doi:10.2307/3605681
[4] Gaver Jr, Donald P. "Observing stochastic processes, and approximate transform inversion." Operations Research 14.3 (1966): 444-459.
[5] Stehfest, Harald. "Algorithm 368: Numerical inversion of Laplace transforms [D5]." Communications of the ACM 13.1 (1970): 47-49.

But yes, Stehfest proposed a discrete version of the Post-Widder formula which is what is being used. 
Thanks for opening this discussion as hopefully it will provide a good resource for those using the algorithm.

**Comment by lrnv** [comment]
Thanks for gathering the bibliography, I completely agree with your choices of citations. Some remarks: 

- From the sources you quoted, we can see that Post and Wilder are talking about the continuous and theoretical case, without the discretisation that your algorithm uses. This discretisation is, 30-35 years later, introduced by Gaver and modified (convergence acceleration) by Stehfest.
- Wikipedia quotes Post and Wilder for the continous inversion formula there : https://en.wikipedia.org/wiki/Inverse_Laplace_transform#Post's_inversion_formula
- French wikipedia (sorry no english translation) quotes Gaver-Stehfest under "Methodes numériques d'inversions" == "Numerical inversion methods", alongside Fourrier's inversion, and only quotes Post-Wilder as a 'Inversion formula' later "for continuous and bounded functions only". https://fr.wikipedia.org/wiki/Transformation_inverse_de_Laplace#M%C3%A9thodes_num%C3%A9riques
- Google scholar gives a lot of results for "Gaver-Stehfest algorithm" and "Post-Wilder formula" but None (zero) for "Post-Wilder algorithm". 
- Kuznetsov, which recently prooved convergence results of the method, uses the 'Gaver-Stehfest' name. Also see References therein; 

I could argue more, but you already understood that I'm voting for a change of name to `gaverstehfest`, `GaverStehfest` or any capitalisation that suits the package naming conventions ;)

**Comment by heltonmc** [comment]
Thanks for your notes! I do agree, and would be happy to make these changes. I think some of the confusion lies maybe in this review paper [1] which mentions the Post-Widder formula under the Gaver-Stehfest method. The papers in my field cite this review article and refer to it as "Post-Widder" which I'm leaning towards perhaps being a mischaracterization :) In fact, it looks like the author is also the author of mpmaths Inverse laplace methods [2] which names it just `stehfest`. At this point it may be best to have one function that dispatches to different methods depending on a keyword or something to preserve some of the namespace. But that is a much bigger change...

[1] Kuhlman, Kristopher L. "Review of inverse Laplace transform algorithms for Laplace-space numerical approaches." Numerical Algorithms 63.2 (2013): 339-355.
[2] https://mpmath.org/doc/1.0.0/calculus/inverselaplace.html

**Comment by heltonmc** [comment]
Thanks @jlapeyre ! I went ahead and made these changes and submitted #25. I've talked with a few other people on this and it seems like "Gaver-Stehfest" is the preferred name which is often how it is referred to in academic papers. I changed the name to `gaverstehfest`. 

This also frees up #10 to be named something like `postwidder` which is a better name fit in the future (maybe with Diffractor.jl when more stable)...

This is my mistake. Thank you everyone.
